Makefile

```diff
@@ -248,3 +248,11 @@ requirements:
 
 dev-requirements:
 	pip install -r requirements/dev.txt
+
+update_redirects:
+	# If files got moved there should be a redirect in redirects.txt
+	sphinx-build -b rediraffewritediff oeps $(BUILDDIR)
```

Comment:
Are both of these really the same command? It just returns a different status code when it finds new redirects?